api/bolt/migrator/migrator.go

```diff
@@ -186,5 +186,13 @@ func (m *Migrator) Migrate() error {
 		}
 	}
 
+	// https://github.com/portainer/portainer/issues/2439
+	if m.currentDBVersion < 15 {
```

Comment:
The database version must be incremented in `portainer.go` to trigger this migration.